transcom/mymove
# B 20654 m25 int

## [B-20654](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A982774)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

Adding the ability to store/save the generated 858s in the system and view them in the admin app

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the test harness scenarios and run 'MakeMoveReadyForEdi'![image](https://github.com/user-attachments/assets/0f48ebb2-8c00-45d8-a359-dc4a82fd3569)
2. grab the payment request number from the payload
![image](https://github.com/user-attachments/assets/7de07825-74c2-4275-b9e2-86f88789811c)
3. run  `go run ./cmd/generate-payment-request-edi/ --payment-request-number 9641-6802-1`
4. see a new record in payment_request_edi_files for that payment_request_number and the edi should match what's showing in terminal output
5. access Admin app
6. select Payment Request Syncada Files link on left nav
7. click the row of  payment request number from above
8. validate the edi string is the same from db and terminal output 
![image](https://github.com/user-attachments/assets/1e08ee0e-8f53-411d-9897-286c4cdb723f)


### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
